Binary files logging-log4j-1.2.9/.build.xml.swp and logging-log4j-1.2.9-new/.build.xml.swp differ
diff -uNr logging-log4j-1.2.9/build.xml logging-log4j-1.2.9-new/build.xml
--- logging-log4j-1.2.9/build.xml	2004-11-01 15:21:14.000000000 -0500
+++ logging-log4j-1.2.9-new/build.xml	2005-05-20 15:10:25.000000000 -0400
@@ -39,7 +39,7 @@
   <property name="jar.dest" value="dist/lib"/>
 
   <!-- The jar file that the jar task will generate -->
-  <property name="jar.filename" value="log4j-${version}.jar"/>
+  <property name="jar.filename" value="log4j-jmx-${version}.jar"/>
 
   <!-- Destination for documentation files -->
   <property name="docs.dest" value="./docs"/>
@@ -307,7 +307,7 @@
      <delete file="${jar.dest}/manifest.mf"/>
   </target>
 
-  <target name="prejar" depends="build">
+  <target name="prejar" depends="build.jmx">
     <mkdir dir="${jar.dest}"/>
     <filter token="version" value="${version}" />
     <copy file="${manifest.src}" tofile="${jar.dest}/manifest.mf"
@@ -325,26 +325,8 @@
     </delete>
 
     <jar jarfile="${jar.dest}/${jar.filename}" basedir="${javac.dest}"
-      includes="${stem}/*.class, ${stem}/xml/log4j.dtd,
-                ${stem}/config/*.class,
-	        ${stem}/helpers/*.class,
-		${stem}/spi/*.class,
-		${stem}/net/*.class,
-		${stem}/jdbc/*.class,
-		${stem}/varia/*.class,
-		${stem}/chainsaw/*.class,
-		${stem}/lf5/**/*.class,
-		${stem}/lf5/**/*.properties,
-		${stem}/lf5/**/*.gif,
-		${stem}/nt/*.class,
-		${stem}/xml/*.class,
-		${stem}/jmx/*.class,
-		${stem}/or/*.class,
-		${stem}/or/sax/*.class,
-                ${stem}/or/jms/*.class,
-		${stem}/config/*.class"
+      includes="${stem}/jmx/*.class"
       excludes="**/UnitTest**"
-      manifest="${jar.dest}/manifest.mf"
     />
   </target>
 
